examples/ami/ami.ipynb

```diff
@@ -0,0 +1,201 @@
+{
```

Comment:
Can you commit the notebook after executing "Run all"? That way, it will serve as extra documentation.